src/material-experimental/mdc-checkbox/_checkbox-theme.scss

```diff
@@ -42,6 +43,11 @@
         ),
       ));
 
+      // MDC should set the disabled color on the label, but doesn't, so we do it here instead.
+      &.mdc-checkbox--disabled label {
```

Comment:
Should we use our own classes (e.g. `mat-mdc-checkbox-disabled`)? Also I don't think that we need the `&.`, the nested selector should be specific enough on its own.

Reply:
Yeah for checkbox that makes sense, radio and slide-toggle didn't have a `mat-mdc-*-disabled` class